site/static/md/reference/attribute_plugins.md

```diff
@@ -95,6 +106,25 @@ Modifiers allow you to modify behavior when merging signals.
 ></div>
 ```
 
+When supplying signals in bulk with object notation, the modifiers still apply to the attribute name:
```

Comment:
What is the purpose of this? It appears to only show the `__ifmissing` modifier, rather than anything related to casing.

Reply:
This section of the docs explains 4 concepts:

1. data-signals-foo
2. data-signals=object notation format
3. case modifiers
4. ifmissing modifier

![image](https://github.com/user-attachments/assets/e8855949-d490-49e4-86b0-df657144d1f6)

---

My addition starting with "When supplying signals.." is directly related to 1, 2, and 4. I don't see why it would have to be only about 3?